src/ExNavigationBar.js

```diff
@@ -240,11 +241,14 @@ export default class ExNavigationBar extends PureComponent {
       });
     });
 
+    let backgroundComponents = scenesProps.map(this._renderBackground, this);
```

Comment:
Every other place skevy or brent has used let !! Still need to change it to const ?